internal/config/mount_config.go

```diff
@@ -71,6 +72,11 @@ type ListConfig struct {
 	EnableEmptyManagedFolders bool `yaml:"enable-empty-managed-folders"`
 }
 
+type GrpcClientConfig struct {
+	// ConnectionPoolSize configures the number of gRPC channel in grpc client.
+	ConnectionPoolSize int `yaml:"conn-pool-size,omitempty"`
```

Comment:
Should we make it Connection**s**  and conn**s**? similar to max-conns-per-host flag